.github/workflows/node.js.yml

```diff
@@ -29,5 +29,4 @@ jobs:
           yarn install --immutable
           yarn build
           yarn test
-          yarn add eslint
           yarn eslint . --ext .js,.jsx,.ts,.tsx
```

Comment:
32 yarn eslint . --ext .js,.jsx,.ts,.tsx
Line 32 will not work when eslint has been removed.